Doc/library/stdtypes.rst

```diff
@@ -4392,6 +4392,22 @@ pairs within braces, for example: ``{'jack': 4098, 'sjoerd': 4127}`` or ``{4098:
          >>> d.values() == d.values()
          False
 
+   .. method:: d | other
```

Comment:
Maybe use ‘describe’ instead of‘method’. (What do the docs for ‘set’ use?)

Reply:
`set` uses `method`, but it lists both the method and operator together. I agree that `describe` is probably better here.